core/node/default_config.yaml

```diff
@@ -22,6 +22,8 @@ baseChain:
     chainId: 31337
     networkUrl: 'http://127.0.0.1:8545'
     blockTimeMs: 2000
+    # TODO: make sure this does not get propagated to production
+    positiveEntitlementCacheTTLSeconds: 5 # Short TTL for testing
 riverChain:
```

Comment:
is this resolved? you just said the opposite thing serge said and then resolved the convo.
also please don't commit todos in this way

Reply:
My mistake - I thought I reverted this file. This file should be unchanged and config-template.yaml changes should stand.

Reply:
I think I said the same thing that Serge said, though - correct me if I'm wrong: default_config is propagated to gamma/omega and config-template is only used for local testing.

Reply:
i was just confused about it being resolved and not reverted. thank you.